App crash on startup due to butterknife

**Actual Behaviour**

App crashes on startup due to `ButterKnife.bind(this);`

**LogCat for the issue**

05-15 02:08:30.269 20920-20920/com.codingblocks.chatter E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.codingblocks.chatter, PID: 20920
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.codingblocks.chatter/com.codingblocks.chatter.AuthenticationActivity}: java.lang.IllegalStateException: Required view 'authentication_web_view' with ID 2131230747 for field 'authenticationWebView' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2665)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6119)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
     Caused by: java.lang.IllegalStateException: Required view 'authentication_web_view' with ID 2131230747 for field 'authenticationWebView' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.
        at butterknife.internal.Utils.findRequiredView(Utils.java:92)
        at butterknife.internal.Utils.findRequiredViewAsType(Utils.java:104)
        at com.codingblocks.chatter.AuthenticationActivity_ViewBinding.<init>(AuthenticationActivity_ViewBinding.java:25)
        at java.lang.reflect.Constructor.newInstance0(Native Method)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
        at butterknife.ButterKnife.createBinding(ButterKnife.java:199)
        at butterknife.ButterKnife.bind(ButterKnife.java:124)
        at com.codingblocks.chatter.AuthenticationActivity.onCreate(AuthenticationActivity.java:39)
        at android.app.Activity.performCreate(Activity.java:6679)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2618)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726) 
        at android.app.ActivityThread.-wrap12(ActivityThread.java) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:154) 
        at android.app.ActivityThread.main(ActivityThread.java:6119) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776) 

--- a/app/src/main/java/com/codingblocks/chatter/AuthenticationActivity.java
+++ b/app/src/main/java/com/codingblocks/chatter/AuthenticationActivity.java
@@ -36,7 +36,6 @@ public class AuthenticationActivity extends AppCompatActivity {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        ButterKnife.bind(this);
 
         // Exclude the title bar for this activity
         requestWindowFeature(Window.FEATURE_NO_TITLE);
@@ -46,6 +45,7 @@ public class AuthenticationActivity extends AppCompatActivity {
         );
 
         setContentView(R.layout.activity_authentication);
+        ButterKnife.bind(this);
 
         // Begin authentication
         authenticationWebView.getSettings().setSupportZoom(true);

